src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(100);
+        ColorSupplier colorSupplier = new ColorSupplier();
```

Comment:
```suggestion
public class Lottery {
    private static final MAX_NUMBER = 100;
    private static final Random random = new Random();
    private static final ColorSupplier colorSupplier = new ColorSupplier();
    
    public Ball getRandomBall() {
        int randomNumber = random.nextInt(MAX_NUMBER);
```
lets improve your code
better avoid magic numbers, most of the time it is better to use constants
better make different suppliers (ColorSupplier, Random)  global